ts/torch_handler/base_handler.py

```diff
@@ -32,35 +30,30 @@ def initialize(self, context):
 
         self.manifest = context.manifest
         properties = context.system_properties
-        model_dir = properties.get("model_dir")
+
         self.device = torch.device("cuda:" + str(properties.get("gpu_id")) if torch.cuda.is_available() else "cpu")
 
-        # Read model serialize/pt file
+        # Read in the model
+        model_dir = properties.get("model_dir")
         serialized_file = self.manifest['model']['serializedFile']
         model_pt_path = os.path.join(model_dir, serialized_file)
+
         if not os.path.isfile(model_pt_path):
             raise RuntimeError("Missing the model.pt file")
 
+        # Torchscript is better, so try to read that first
+        is_torchscript = True
         try:
             logger.debug('Loading torchscript model')
-            self.model = torch.jit.load(model_pt_path)
+            self.model = self._load_torchscript_model(model_pt_path)
         except RuntimeError as e:
-            # Read model definition file
+            is_torchscript = False
+
+        if not is_torchscript:
```

Comment:
This seems redundant. Can we remove it?

Reply:
Fixed. I'll merge in the other PR when this branch merges in.